src/pretalx/settings.py

```diff
@@ -405,6 +421,9 @@ def reduce_dict(data):
 
 MESSAGE_STORAGE = 'django.contrib.messages.storage.session.SessionStorage'
 
+emails = config.get('logging', 'email', fallback='').split(',')
+ADMIN = [(email, email) for email in emails if email]
```

Comment:
since we split those values into a list, trimming them would be nice in case someone sets a value like `email@one.com, admin@google.com`.